### DISCUSSION TITLE
Why not have the settings also be able to come from the job file?

### DISCUSSION THREAD

**Discussion by Marcuzzz** [inlife/nexrender#759] [Ideas]
Hi,

I was curious why you don't have this line in:
```settings = Object.assign(settings, parsedJob['settings']);```

Just before:

```render(parsedJob, settings)...```

In the packages/nexrender-cli/src/bin.js

**Comment by inlife** [comment]
Good question! I do not know anymore. 😄 

Do you wish to contribute to this as an improvement?

**Reply by Marcuzzz** [reply]
Sure I can contribute this improvement. I have it available as commit... 
How do you want me to push it to the repo?

**Reply by inlife** [reply]
Pull request would be the best!
